### DISCUSSION TITLE
getBoundingRect() function does not calculate size of object on mobile device.

### DISCUSSION THREAD

**Discussion by vipin-vidyut** [fabricjs/fabric.js#8459] [General]
I am using Fabric Js, I am trying to scale all canvas object to screen. if all object is smaller than viewport no need to scale but if object width is greater than viewport we need of scale it. for this I am doing 
`var objs = canvas.getObjects();
canvas.discardActiveObject();
sel = new fabric.ActiveSelection(objs, {});
 var obj = sel.getBoundingRect();
 var w = canvas.width;
 var h = canvas.height;
if(w < obj.width){
   sel.scaleToWidth(Number(w -50));
   canvas.requestRenderAll();
}
` 
This is working on Desktop But not on Mobile device.

**Comment by ShaMan123** [comment]
add a repro
